tests/helpers/preload-assets.js

```diff
@@ -0,0 +1,26 @@
+import Ember from 'ember';
+import AssetLoader from 'ember-asset-loader/services/asset-loader';
```

Comment:
It seems a bit odd to import the service and create an instance instead of injecting it and using the same instance. 

I'm worried this could bite us later if someone is explicitly preloading assets, but we're using a different [cache](https://github.com/trentmwillis/ember-asset-loader/blob/master/addon/services/asset-loader.js#L176)